sd-release.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -x
+
+MVN=$(type -p mvn)
+MVN_FLAGS="-B"
+
+if [ -n "${WORKSPACE}" ]; then
+  MVN_FLAGS="${MVN_FLAGS} -f ${WORKSPACE}/pom.xml"
+  MVN_FLAGS="${MVN_FLAGS} -Dmaven.repo.local=${WORKSPACE}/.repository"
+fi
+
+# Set the version to the next release
+${MVN} ${MVN_FLAGS} \
+  versions:set \
+    -DnewVersion=${KITE_RELEASE_VERSION} \
```

Comment:
They're expected to come from the Jenkins job. I'll add a check that they're not empty at the top of the script and then push.